### ISSUE TITLE
Autocomplete predictions

### ISSUE THREAD

**Issue by OscarLarm** [annk15/MTGKeywordHelper#16]
We should look into making a autocomplete predictions feature for the search bar, to help users find the right keyword easier and faster.

**Comment by annk15** [comment]
I will use Jquery and see if I can make this work using the following script.

```

<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>

<input id="search" type="text" placeholder="Search...">

<script>
  $(function() {
    var availableTags = ["Apple", "Banana", "Cherry", "Date", "Fig", "Grape"];
    $("#search").autocomplete({
      source: availableTags
    });
  });
</script>

```

**Comment by annk15** [comment]
Fixed in #45
